ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/ethernet-interfaces.j2

```diff
@@ -28,6 +28,11 @@ interface {{ ethernet_interface.name }}
    no shutdown
 {%     endif %}
 {%     if print_ethernet is arista.avd.defined(true) %}
+{%         if ethernet_interface.load_interval is arista.avd.defined %}
```

Comment:
Should this be: 
`{% if ethernet_interface.load_interval is arista.avd.defined and ethernet_interface.load_interval != load_interval.default %}`

Reply:
We should not implement logic like this in eos_cli_config_gen. If you set the value, it will be configured. I am working on a PR to remove all the print_ethernet stuff as well.

Reply:
This feature should _not_ be inside a `print_ethernet` clause. We want it to be configured if set - even for port-channel members.

Reply:
removed print ethernet clause